blockonomics-woocommerce.php

```diff
@@ -60,11 +60,24 @@ function blockonomics_woocommerce_init()
     add_action('init', 'load_plugin_translations');
     add_action('woocommerce_order_details_after_order_table', 'nolo_custom_field_display_cust_order_meta', 10, 1);
     add_action('woocommerce_email_customer_details', 'nolo_bnomics_woocommerce_email_customer_details', 10, 1);
+    add_action('admin_enqueue_scripts', 'blockonomics_load_admin_scripts' );
     add_action( 'restrict_manage_posts', 'filter_orders' , 20 );
     add_filter( 'request', 'filter_orders_by_address_or_txid' );	
     add_filter('woocommerce_payment_gateways', 'woocommerce_add_blockonomics_gateway');
     add_filter('clean_url', 'bnomics_async_scripts', 11, 1 );
 
+
+    /**
+     * Add Styles to Blockonomics Admin Page
+     **/
+    function blockonomics_load_admin_scripts(){ 
+        $current_screen = get_current_screen();
+        if ( strpos($current_screen->base, 'settings_page_blockonomics_options') === false) {
```

Comment:
admin_enqueue_scripts has a $hook parameter that can be used. `if ( $hook === 'settings_page_blockonomics_options' )`

Reply:
Awesome, that makes a lot of sense and is a much better solution. Thanks! I've pushed the changes